doc/release-notes/release-4.sgml

```diff
@@ -492,6 +510,10 @@ This section gives an account of those changes in three categories:
 	   actually auto-detect them without configuring the module
 	   list manually.
 
+	<tag>--enable-esi</tag>
+	<p>Custom ESI parser has been removed.
+	   Libxml2 or libexpat are now required to enable ESI processor.
```

Comment:
s/are/is/ or see a version mentioned earlier that does not require spelling libxml2 as `Libxml2`.

I would also s/processor/processing/.